### PR TITLE
#166880068 setup environment for writing tests

### PR DESCRIPTION
**Pull request by frankopkusianwar**
#### What does this PR do?

setup an environment for writing tests

#### Description of Task to be completed?

- add jest testing framework 
- add a javascript testing utility enzyme
- add a test to check the setup functionality

#### How should this be manually tested?

- Clone this repo and switch to this branch
- `cd` into the root of the project directory and run `npm install` to install the packages
  in the package.json file.
-  Run `npm test` to check the setup functionality.
- There should be a passing test signifying success

#### What are the relevant pivotal tracker stories?

[#166880068](https://www.pivotaltracker.com/story/show/166880068)

#### Screenshots (if appropriate)

tests

### CODE REVIEW

**Review by richien — APPROVED** [review]
Hi @frankopkusianwar, I have tested this locally and it looks ok to me. Good work.

**Review by NicholusMuwonge — APPROVED** [review]
Works as expected locally and I found no problem with your code implementation. Thanks for the good work.

**Review by sebbss — APPROVED** [review]

File: src/__test__/App.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../components/App';
+
+
+describe('App', () => {
+  test('should match snapshot', () => {
```
Comment (MuhanguziDavid) [inline src/__test__/App.test.js]: use the word `it` instead of `test` for each test i.e.
`it(your test)`

Comment (frankopkusianwar) [reply src/__test__/App.test.js]: @MuhanguziDavid this has been resolved

Comment (MuhanguziDavid) [reply src/__test__/App.test.js]: @frankopkusianwar , I meant that you needed to change the word `test` to `it` literally changing only that word. Leave the rest as it was.

**Pull request merged**